### DISCUSSION TITLE
[Idea]: Better release management

### DISCUSSION THREAD

**Discussion by mtdvlpr** [sircharlo/meeting-media-manager#3652] [Ideas] [answered]
### Is your idea related to a problem?

When users install a test build with a preliminary version of `v24.12.1` before the actual `v24.12.1` is released and then we release the real `v24.12.1`, they won't update to the official release, because they're already on the latest version.

### Describe your idea

Changing the release cycle to only release even numbered patch releases: 24.12.0, 24.12.2, 24.12.4, etc. and reserve the odd numbers for test builds.

After each release, we bump the version to an odd numbered version (24.12.1, 24.12.3, etc.). Then users might be asked to install the odd numbered test build or they install it themselves (it's publicly available). And when we're ready for the next release, instead of releasing the odd numbered version, we bump the version again to an even number.

This will help end-users to always correctly update to official releases when available, it will help us to differentiate users running test builds / official releases and it will help organize Sentry data into test vs. official releases/issues.

### Describe alternatives you've considered

Alternatively, we can also bump the version after a release to a beta release (e.g. `v24.12.1-beta.0`). That would allow us to re-introduce automatic beta releases.

### Confirmations

- [X] I confirm that I have read the documentation website.
- [X] I confirm that I have searched through the existing issues to make sure my idea is not a duplicate.
- [X] I confirm that I have searched through the existing discussions to make sure my idea is not a duplicate.

**Comment by mtdvlpr** [comment]
@sircharlo, what do you think?

**Reply by sircharlo** [reply]
I wonder if we could simply do like before, have automatic beta releases. That seemed to work nicely in the past.

**Reply by mtdvlpr** [reply, marked as answer]
I've re-introduced the beta release workflow. I'll implement an advanced setting to automatically receive these beta updates another time.

**Reply by sircharlo** [reply]
Awesome! Thank you :)

**Reply by e-Feitosa** [reply]
I downloaded the beta version, and nothing worked. The program installs, but none of the settings open, and it also doesn't close (terminate). Another issue, the beta version is available for everyone to download from the main link, I'm not sure if that's the intention. A suggestion is to make the beta version like the previous one where there was an option to activate the beta version in M3 and with that warning not to use it in KH.

![image](https://github.com/user-attachments/assets/e5323410-6ef9-4b52-a4e0-edaaa4ce1204)

**Reply by mtdvlpr** [reply]
@e-Feitosa, thanks! The issues with the current beta release have been resolved and will be included in the next beta release. The docs no longer point to beta releases and a feature to let users automatically update to beta releases has been developed in #3720.

**Reply by e-Feitosa** [reply]
@mtdvlpr, It will always be a pleasure to contribute. I deeply appreciate the valuable work of all of you.
